src/Bindings/DeprecatedBindings.cpp

```diff
@@ -347,6 +347,41 @@ static int tolua_set_cBlockInfo_m_PlaceSound(lua_State * tolua_S)
 
 
 
+static int tolua_get_cBlockInfo_m_IsSnowable(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+	if (!L.CheckParamSelf("const cBlockInfo"))
+	{
+		return 0;
+	}
+
+	L.Push("");
```

Comment:
This should at the very least be returning a boolean. Even better would be to have this return `cBlockInfo::IsSnowable()`.  Possibly by storing the block id in the `cBlockInfo` instance without exporting it to lua.

Reply:
So would returning false (or true) be good enough, because with my limited knowledge of C++ I can't find a decent way to store the block id in the cBlockInfo instance.

Reply:
Do you mind if I just make the change myself?

Reply:
I don't mind at all. 👍